### ISSUE TITLE
tmsh modify sys httpd ssl-ciphersuite <new cipher>

### ISSUE THREAD

**Issue by jackyatesaustin** [F5Networks/f5-declarative-onboarding#178]
### Is your feature request related to a problem? Please describe.
Right now, using DO you can only use a cipher suite that are in the predefined list. Customer needs to add a cipher suite

### Describe the solution you'd like
For this to be available via DO

### Describe alternatives you've considered
None, hoping to move onboarding process to DO.

**Comment by dstokesf5** [comment]
Which cipher does the customer need, and would adding it to the cipher property of the SSHD class be sufficient for your use case?

**Comment by dstokesf5** [comment]
I misread this initially as needing to configure SSHD instead of HTTPD.

There are several cipher suite options available in DO through the HTTPD.sslCipherSuite property:
```
                                "ECDHE-RSA-AES256-GCM-SHA384",
                                "ECDHE-ECDSA-AES256-GCM-SHA384",
                                "ECDHE-RSA-AES256-SHA384",
                                "ECDHE-ECDSA-AES256-SHA384",
                                "ECDHE-RSA-AES256-SHA",
                                "ECDHE-ECDSA-AES256-SHA",
                                "DH-DSS-AES256-GCM-SHA384",
                                "DHE-DSS-AES256-GCM-SHA384",
                                "DH-RSA-AES256-GCM-SHA384",
                                "DHE-RSA-AES256-GCM-SHA384",
                                "DHE-RSA-AES256-SHA256",
                                "DHE-DSS-AES256-SHA256",
                                "DH-RSA-AES256-SHA256",
                                "DH-DSS-AES256-SHA256",
                                "DHE-RSA-AES256-SHA",
                                "DHE-DSS-AES256-SHA",
                                "DH-RSA-AES256-SHA",
                                "DH-DSS-AES256-SHA",
                                "DHE-RSA-CAMELLIA256-SHA",
                                "DHE-DSS-CAMELLIA256-SHA",
                                "DH-RSA-CAMELLIA256-SHA",
                                "DH-DSS-CAMELLIA256-SHA",
                                "ECDH-RSA-AES256-GCM-SHA384",
                                "ECDH-ECDSA-AES256-GCM-SHA384",
                                "ECDH-RSA-AES256-SHA384",
                                "ECDH-ECDSA-AES256-SHA384",
                                "ECDH-RSA-AES256-SHA",
                                "ECDH-ECDSA-AES256-SHA",
                                "AES256-GCM-SHA384",
                                "AES256-SHA256",
                                "AES256-SHA",
                                "CAMELLIA256-SHA",
                                "PSK-AES256-CBC-SHA",
                                "ECDHE-RSA-AES128-GCM-SHA256",
                                "ECDHE-ECDSA-AES128-GCM-SHA256",
                                "ECDHE-RSA-AES128-SHA256",
                                "ECDHE-ECDSA-AES128-SHA256",
                                "ECDHE-RSA-AES128-SHA",
                                "ECDHE-ECDSA-AES128-SHA",
                                "DH-DSS-AES128-GCM-SHA256",
                                "DHE-DSS-AES128-GCM-SHA256",
                                "DH-RSA-AES128-GCM-SHA256",
                                "DHE-RSA-AES128-GCM-SHA256",
                                "DHE-RSA-AES128-SHA256",
                                "DHE-DSS-AES128-SHA256",
                                "DH-RSA-AES128-SHA256",
                                "DH-DSS-AES128-SHA256",
                                "DHE-RSA-AES128-SHA",
                                "DHE-DSS-AES128-SHA",
                                "DH-RSA-AES128-SHA",
                                "DH-DSS-AES128-SHA",
                                "DHE-RSA-SEED-SHA",
                                "DHE-DSS-SEED-SHA",
                                "DH-RSA-SEED-SHA",
                                "DH-DSS-SEED-SHA",
                                "DHE-RSA-CAMELLIA128-SHA",
                                "DHE-DSS-CAMELLIA128-SHA",
                                "DH-RSA-CAMELLIA128-SHA",
                                "DH-DSS-CAMELLIA128-SHA",
                                "ECDH-RSA-AES128-GCM-SHA256",
                                "ECDH-ECDSA-AES128-GCM-SHA256",
                                "ECDH-RSA-AES128-SHA256",
                                "ECDH-ECDSA-AES128-SHA256",
                                "ECDH-RSA-AES128-SHA",
                                "ECDH-ECDSA-AES128-SHA",
                                "AES128-GCM-SHA256",
                                "AES128-SHA256",
                                "AES128-SHA",
                                "SEED-SHA",
                                "CAMELLIA128-SHA",
                                "PSK-AES128-CBC-SHA",
                                "ECDHE-RSA-RC4-SHA",
                                "ECDHE-ECDSA-RC4-SHA",
                                "ECDH-RSA-RC4-SHA",
                                "ECDH-ECDSA-RC4-SHA",
                                "RC4-SHA",
                                "RC4-MD5",
                                "PSK-RC4-SHA",
                                "ECDHE-RSA-DES-CBC3-SHA",
                                "ECDHE-ECDSA-DES-CBC3-SHA",
                                "EDH-RSA-DES-CBC3-SHA",
                                "EDH-DSS-DES-CBC3-SHA",
                                "DH-RSA-DES-CBC3-SHA",
                                "DH-DSS-DES-CBC3-SHA",
                                "ECDH-RSA-DES-CBC3-SHA",
                                "ECDH-ECDSA-DES-CBC3-SHA",
                                "DES-CBC3-SHA",
                                "PSK-3DES-EDE-CBC-SHA",
                                "EDH-RSA-DES-CBC-SHA",
                                "EDH-DSS-DES-CBC-SHA",
                                "DES-CBC-SHA",
                                "EXP-EDH-RSA-DES-CBC-SHA",
                                "EXP-EDH-DSS-DES-CBC-SHA",
                                "EXP-DES-CBC-SHA",
                                "EXP-RC2-CBC-MD5",
                                "EXP-RC4-MD5"
```

I am reviewing the string you sent me on Team to determine if that can be configured with existing DO options.

**Comment by jackyatesaustin** [comment]
@dstokesf5 thank you!

**Comment by dstokesf5** [comment]
Thank you for your feedback. Your request has been added to our internal product backlog as AUTOTOOL-2759.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
